lib/node_modules/@stdlib/_tools/eslint/rules/doctest-annotation-spacing/lib/defaults.json

```diff
@@ -0,0 +1,3 @@
+{
+  "numSpaces": 1
```

Comment:
@Planeshifter Were you to support space configuration before and after keyword: `beforeSpaces` and `afterSpaces`.